commercetools/resource_shipping_zone_rate.go

```diff
@@ -264,28 +283,43 @@ func unmarshallShippingRatePriceTiers(d *schema.ResourceData) ([]platform.Shippi
 	for _, priceTier := range values.([]interface{}) {
 		tierMap := priceTier.(map[string]interface{})
 
-		priceMap := tierMap["price"].([]interface{})[0].(map[string]interface{})
-		price := platform.Money{
-			CurrencyCode: priceMap["currency_code"].(string),
-			CentAmount:   priceMap["cent_amount"].(int),
+		var price *platform.Money
+		priceList := tierMap["price"].([]interface{})
+		if len(priceList) > 0 {
```

Comment:
There is a utility function available to achieve this:

`priceMap, err := elementFromSlice(priceTier, "price")`